datafusion/core/src/catalog/schema.rs

```diff
@@ -285,19 +165,23 @@ mod tests {
         let filename = format!("{}/{}", testdata, "alltypes_plain.parquet");
         let table_path = ListingTableUrl::parse(filename).unwrap();
 
-        let schema = ObjectStoreSchemaProvider::new();
-        let _store = schema.register_object_store("test", Arc::new(LocalFileSystem {}));
+        let catalog = MemoryCatalogProvider::new();
```

Comment:
Here we can see a MemoryCatalogProvider being used in place of ObjectStoreSchemaProvider